graphql/handler/transport/websocket.go

```diff
@@ -27,6 +27,7 @@ type (
 		CloseFunc             WebsocketCloseFunc
 		KeepAlivePingInterval time.Duration
 		PingPongInterval      time.Duration
+		MissingPongOk         bool
```

Comment:
Maybe a doc string? Is the intent disable the ping/pong feature? Maybe a different name?